### ISSUE TITLE
Qualitative questions with correct answers

### ISSUE THREAD

**Issue by dsanson** [Carnap/Carnap#155]
I'd like to have a way to specify the correct answer to a qualitative short answer question, maybe with the ability to specify a few variants. The idea would be to use this for "fill in the blank" type questions where it was reasonable to expect a specific word or short phrase, and perhaps also for (very rigidly graded) translation questions, where students are asked to follow a precise algorithm to render a symbolic sentence into a canonical English form.

**Comment by gleachkr** [comment]
Brainstorming: how should answers be normalized in order to prevent false negatives? Two and a half ideas

1. collapse inter-word whitespace, remove leading and trailing whitespace
2. lower-case all letters
3. remove punctuation (?)

**Comment by dsanson** [comment]
Yes to (1) and (2). (3) seems like it could be problematic in some use-cases and very useful for others...
